FIRO_TSEnsembles/src/test/java/hec/ensemble/DatabaseGenerator.java

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
Looks like something else to move to test.

Reply:
that is in test.